src/Sulu/Bundle/ContentBundle/Resources/views/Template/content-types/block.html.twig

```diff
@@ -5,7 +5,7 @@
             <div id="edit-text-blocks-{{ id|raw }}" tabindex="0" class="btn link text-blocks-toggle-btn pull-right hidden"><%= translate('sulu.content.edit.blocks') %></div>
             <div class="clear"></div>
         </div>
-        <div class="sortable grid-row"
```

Comment:
Are you sure that you can remove this class? I don't think so, at least the blocks are not sortable anymore.